modules/gapi/include/opencv2/gapi/own/exports.hpp

```diff
@@ -8,11 +8,18 @@
 #ifndef OPENCV_GAPI_OWN_TYPES_HPP
 #define OPENCV_GAPI_OWN_TYPES_HPP
 
+#   define GAPI_WRAP
+
 #   if defined(__OPENCV_BUILD)
 #       include <opencv2/core/base.hpp>
 #       define GAPI_EXPORTS CV_EXPORTS
+        /* special informative macros for wrapper generators */
+#       define GAPI_EXPORTS_W_SIMPLE CV_EXPORTS
+#       define GAPI_EXPORTS_W CV_EXPORTS
```

Comment:
Maybe it's more safe to define in this way?
```
#       define GAPI_EXPORTS_W_SIMPLE CV_EXPORTS_W_SIMPLE
#       define GAPI_EXPORTS_W CV_EXPORTS_W
```

Reply:
I thinks so, thanks

Reply:
Fixed